sccm/apps/deploy-use/import-export-applications.md

```diff
@@ -0,0 +1,46 @@
+---
+
+# Mandatory fields.
+title: Import/Export Applications
+titleSuffix: Configuration Manager
+description: Import and export applications.
+author: chriskibble
+ms.author: NO VALUE SET # Microsoft employees only
+ms.date: 5/6/2019
+ms.topic: conceptual
+ms.technology: configmgr-app
+---
+# Create applications in Configuration Manager
+
+*Applies to: System Center Configuration Manager (Current Branch)*
+
+Configuration Manager allows for the import and export of applications within the console.  This allows you to copy applications in to other environments, such as copying from a test environment into a production environment.
+
+## <a name="bkmk_export"></a> Export an Application
+
+1. Within the Configuration Manager console, right click on the application to export and select **Export**.
```

Comment:
Avoid use of the verb "click" as it's not inclusive (it implies the ability to use a mouse button). For more information, see [click in the style guide](https://docs.microsoft.com/style-guide/a-z-word-list-term-collections/c/click).

```suggestion
1. In the Configuration Manager console, select the application to export. In the ribbon, choose **Export**.
```

Reply:
Fixed in #1585